tests/model_enums/tests.py

```diff
@@ -143,6 +143,12 @@ class Fruit(models.IntegerChoices):
                 APPLE = 1, 'Apple'
                 PINEAPPLE = 1, 'Pineapple'
 
+    def test_str(self):
+        for test in [Suit, YearInSchool, Vehicle, Gender]:
```

Comment:
Tuple here?

Reply:
I went for list because it's not a _record_, an _unnamedtuple™_ with different fields, but a straight list of different items I want to check.